### DISCUSSION TITLE
Does Oceananigans support coastal modelling?

### DISCUSSION THREAD

**Discussion by stanleesocca** [CliMA/Oceananigans.jl#3272] [Computational science!]
Hi devs. This is not an issue but more of comment. I will immediately close this after some feedbacks. 

I was wondering if Oceananigans in its current form has support for coastal or regional modelling? I ask because my impression is that Oceananigans follows similar paradigm as MITgcm with the latter seemingly used in regional ocean modelling. Are there plans to use the model for such setup? What will it takes for the community to get to such level? 

Thanks.

**Comment by simone-silvestri** [comment]
Hi @stanleesocca, if you're ok we can continue this discussion in the discussion section and I will close this issue

**Comment by glwagner** [comment]
Just convert to discussion?

**Comment by stanleesocca** [comment]
Yes please do @simone-silvestri @glwagner
